common/main/he.xml

```diff
@@ -8636,13 +8636,13 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<pattern type="range">{0}–{1}</pattern>
 		</miscPatterns>
 		<minimalPairs>
-			<pluralMinimalPairs count="one">שנה</pluralMinimalPairs>
-			<pluralMinimalPairs count="two">שנתיים</pluralMinimalPairs>
+			<pluralMinimalPairs count="one">השנה האחרונה הייתה קשה</pluralMinimalPairs>
+			<pluralMinimalPairs count="two">השנתיים האחרונות היו קשות</pluralMinimalPairs>
 			<pluralMinimalPairs count="many">{0} שנה</pluralMinimalPairs>
-			<pluralMinimalPairs count="other">{0} שנים</pluralMinimalPairs>
+			<pluralMinimalPairs count="other">{0} השנים האחרונות היו קשות</pluralMinimalPairs>
```

Comment:
Isn't 'שנים' the word for year?

Reply:
Yes it is. I didn't change the 'many' case because they didn't come back with the form needed, and it looks likely we'll remove that.

Reply:
No I meant in the 'other' line. I see the word for year in the phrase or am I misunderstanding? I don't really know how to read hebrew?